persistence/javadsl/src/main/scala/com/lightbend/lagom/javadsl/persistence/PersistentEntity.scala

```diff
@@ -211,8 +211,10 @@ abstract class PersistentEntity[Command, Event, State] {
     private var commandHandlers: Map[Class[_ <: Command], JBiFunction[_ <: Command, CommandContext[Any], Persist[_ <: Event]]] =
       cmdHandlers
 
+    @Deprecated
```

Comment:
@ayushprashar, since this is a Scala file, we should use the `scala.deprecated` annotation instead.

Reply:
@renatocaval sure, made the changes! Please have a look.

Reply:
I encountered some issues, closing this PR and raising a new one.
@renatocaval